aiida_ase/calculations/ase.py

```diff
@@ -278,7 +283,7 @@ def return_a_function(v):
         calcinfo.remote_copy_list = remote_copy_list
 
         codeinfo = common.CodeInfo()
-        codeinfo.cmdline_params = [self._INPUT_FILE_NAME]
+        codeinfo.cmdline_params = [settings.pop('CMDLINE', []), self._INPUT_FILE_NAME]
```

Comment:
The `CMDLINE` key should really be a list as multiple command line arguments should be able to be passed.
```suggestion
        cmdline_params = settings.pop('CMDLINE', [])
        if not isinstance(cmdline_params, (list, tuple)):
            cmdline_params = list(cmdline_params)
        codeinfo.cmdline_params = cmdline_params.extend(self._INPUT_FILE_NAME)
```